reference_data/platforms.yaml

```diff
@@ -201,3 +201,180 @@
       debug: true
       remote_sync_policy: on_demand
       repository_sync_policy: additive
+
+- name: AlmaLinux-9-beta
+  distr_type: rhel
+  distr_version: '9'
+  test_dist_name: almalinux
+  modularity:
+    modified_packages_url: 'https://git.almalinux.org/almalinux/modified_packages/raw/branch/main/modified_packages.yml'
+    packages_git: 'https://git.almalinux.org/rpms/'
+    git_tag_prefix:
+      modified: a9-beta
+      non_modified: c9-beta
+    versions:
+      - name: '9'
+        version_prefix: '90000'
+        dist_prefix: 'el9.0.0'
+  type: rpm
+  arch_list:
+    - x86_64
+    - aarch64
+    - ppc64le
+    - s390
```

Comment:
2 things:
1. We need i686 arch anyway since we have multilib
2. I think `s390` is called `s390x`